### ISSUE TITLE
Connect My Computer: Agent cleanup daemon exits immediately after launch

### ISSUE THREAD

**Issue by ravicious** [gravitational/teleport#36865]
…which means we stop the agent immediately as well.

It's likely due to changes in Electron v28. I'm able to reproduce it locally with a signed build.

**Comment by ravicious** [comment]
After adding `stdio: 'inherit'` when forking the cleanup daemon I see this:

```
[0118/173102.419983:ERROR:node_main.cc(129)] Node.js environment variables are disabled because this process is invoked by other apps.
/Users/rav/src/teleport/web/packages/teleterm/build/release/mac-arm64/Teleport Connect.app/Contents/Resources/app.asar/build/app/dist/main/agentCleanupDaemon.js:13202
  throw new Error('Agent cleanup daemon must be spawned from a Node.js parent that establishes an IPC channel');
  ^

Error: Agent cleanup daemon must be spawned from a Node.js parent that establishes an IPC channel
    at /Users/rav/src/teleport/web/packages/teleterm/build/release/mac-arm64/Teleport Connect.app/Contents/Resources/app.asar/build/app/dist/main/agentCleanupDaemon.js:13202:9
    at /Users/rav/src/teleport/web/packages/teleterm/build/release/mac-arm64/Teleport Connect.app/Contents/Resources/app.asar/build/app/dist/main/agentCleanupDaemon.js:13301:3
    at Object.<anonymous> (/Users/rav/src/teleport/web/packages/teleterm/build/release/mac-arm64/Teleport Connect.app/Contents/Resources/app.asar/build/app/dist/main/agentCleanupDaemon.js:13303:12)
    at Module._compile (node:internal/modules/cjs/loader:1271:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1326:10)
    at Module.load (node:internal/modules/cjs/loader:1126:32)
    at Module._load (node:internal/modules/cjs/loader:967:12)
    at l._load (node:electron/js2c/asar_bundle:2:13642)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:101:12)
    at node:internal/main/run_main_module:23:47

Node.js v18.18.2
```

(I'll have to make it so that those failures are easier to see)

**Comment by ravicious** [comment]
This is caused by `process.send` not being available in the cleanup daemon. https://github.com/electron/electron/issues/40987

**Comment by ravicious** [comment]
Downgrading to 28.1.1 for now to unblock release. If this get's deemed as WONTFIX, I'll open another issue.

**Comment by ravicious** [comment]
The Electron team is planning to fix this, so we should be safe to upgrade whenever the fix lands.
